src/utils/font-size.js

```diff
@@ -0,0 +1,80 @@
+import styles from '@/styles/index.scss'
+// TODO: const styles = { fontRootSize: '16px' }
```

Comment:
This is a blocker at the moment. Interestingly, it works with Vue + webpack. You can import `.js` and `.scss` files with no issues. Except that if you run this file with Cypress in the e2e tests, then it complains that it cannot file any tests in your file. With the commented code I was able to get it to run, but it would be preferable to get the value from Vuetify, in case that changes in future releases.